spec/features/invitations_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+feature "Sending invitations" do
+  scenario "user sends invitation to friend, friend opens the email and registers" do
+    bob = Fabricate(:user, full_name: "Bob Robertson")
+    clear_emails
+
+    sign_in(bob)
+    send_invitation_to_friend
```

Comment:
I would be explicit here about who is `bob` inviting. I see you are asserting below about "Kim Possible". In your `check_if_user_add_to_following("Bob Robertson")` You are checking on who is follower.